apps/src/turtle/blocks.js

```diff
@@ -259,35 +259,84 @@ exports.install = function (blockly, blockInstallOptions) {
     }
   };
 
-blockly.Blocks.point_to = {
-    // Block for pointing to a specified direction
-    helpUrl: '',
-    init: function () {
-      this.setHSV(184, 1.00, 0.74);
-      this.appendDummyInput()
-          .appendTitle(msg.pointTo());
-      this.appendDummyInput()
-          .appendTitle(new blockly.FieldTextInput('0', blockly.FieldTextInput.numberValidator), 'DIRECTION')
+  generator.draw_turn_inline = function () {
+    // Generate JavaScript for turning left or right.
+    var value = window.parseFloat(this.getTitleValue('VALUE'));
+    return 'Turtle.' + this.getTitleValue('DIR') +
+        '(' + value + ', \'block_id_' + this.id + '\');\n';
```

Comment:
Nit: can we use string interpolation here?

Reply:
Yes.  Is string interpolation faster than string concatenation?  Is the benefit of string interpolation the use of fewer lines of code than string concatenation?

Reply:
Mainly code clarity!  It may actually be a bit slower than concatenation (depending on the impl) but we're optimizing for developer time over CPU time in general, unless we specifically need to speed up some part of the codebase.